ci/ray_ci/ray_docker_container.py

```diff
@@ -49,6 +49,8 @@ def run(self) -> None:
                 "python .buildkite/copy_files.py --destination docker_login",
             ]
             for alias in self._get_image_names():
+                if os.environ["RAYCI_SCHEDULE"] != "NIGHTLY" and "nightly" in alias:
```

Comment:
the value of `RAYCI_SCHEDULE` should be `nightly`, rather than `NIGHTLY`. we do not need to use uppercase for env var values.

Reply:
fixed